tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -137,7 +137,7 @@ def _sync_rbds(self):
                     size=v['size'],
                     pool_id=pool_id,
                     flags=v['flags'],
-                    provisioned=self._to_bytes(v['provisioned']),
+                    provisioned=self._to_bytes(v.get("provisioned", 0)),
```

Comment:
I had given it 0 as here in definition, data type of provisioned shows int (https://github.com/Tendrl/ceph-integration/blob/develop/tendrl/ceph_integration/objects/definition/ceph.yaml#L271) . Should I put it as None ? @shtripat @nthomas-redhat @GowthamShanmugam